src/pool/srv_target.c

```diff
@@ -1075,22 +1075,49 @@ cont_iter_cb(uuid_t cont_uuid, vos_iter_entry_t *ent, void *data)
 	return arg->callback(cont_uuid, ent, arg->arg);
 }
 
+struct dtx_resync_args {
+	daos_handle_t	ph;
+	uuid_t		po_uuid;
+	uuid_t		co_uuid;
+	uint32_t	ver;
+};
+
+static int
+dtx_resync_ult(void *data)
+{
+	struct dtx_resync_args	*args = data;
+	int			 rc;
+
+	rc = dtx_resync(args->ph, args->po_uuid, args->co_uuid, args->ver,
```

Comment:
If there is already another ULT is resyncing the container, dtx_resync() will run into a loop in 'block' mode, which means this high priority ULT will be frequently (or always) scheduled but do nothing?

Reply:
yes, I think it is possible, but only happens in node failure then rebuild mode.
In the scheduler there is still 20% chance to schedule other ULT tasks if there is ULT in URGENT pool.